packages/jaeger-ui/src/components/common/vars.css

```diff
@@ -8,18 +8,25 @@ SPDX-License-Identifier: Apache-2.0
  * 
  * This file contains all color and layout variables used throughout the application.
  * New naming scheme follows semantic conventions for better maintainability.
+ * 
+ * Variables are defined on both body and .ant-layout to ensure they're available
+ * for fixed-position elements and components rendered outside the Layout.
+ * 
+ * Light mode uses fallback values since --ant-* variables may not be available
+ * on body element. Dark mode provides explicit overrides.
```

Comment:
yes, this was quite a fight yesterday to get working, but sounds like it still has gaps. What are the examples of "fixed-position elements and components rendered outside the Layout"?

Reply:
I think #3247 fixes that issue by making both `--ant-xxx` and our custom variables visible at the `:root` level.